compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -4,6 +4,7 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <cstdint>
```

Comment:
the usual practice around here is to rely on MLIR headers to include this for us.

Reply:
VSCode is now very smart to auto include this header file for me without notifying...

Reply:
I actually have the same problem. If you ever find how to disable that, please share.